site/docs/plugins/conversations.md

```diff
@@ -572,6 +572,68 @@ async function waitForMe(conversation, ctx) {
 An update can mean that a text message was sent, or a button was pressed, or something was edited, or virtually any other action was performed by the user.
 Check out the full list in the Telegram docs [here](https://core.telegram.org/bots/api#update).
 
+The `wait` method always yields a new [context object](../guide/context.md) representing the received update.
+That means you're always dealing with as many context objects as there are updates received during the conversation.
+
+<CodeGroup>
+<CodeGroupItem title="TypeScript" active>
+
+```ts
+const TEAM_REVIEW_CHAT = -10018121912;
```

Comment:
Can you tell us who this ID belongs to? I understand that chat IDs are not private information and can be shared publicly, but would it be better for us to replace it with an obvious random chat ID, like -9991234567890, or even better, promote our Telegram channel, like -1001402419718, which is the ID for the grammY News?

Reply:
Good idea!